convert2rhel/breadcrumbs.py

```diff
@@ -204,6 +188,8 @@ def write_obj_to_array_json(path, new_object, key):
         with open(path, "a") as file:
             file_content = {key: []}
             json.dump(file_content, file, indent=4)
+            # the file can be changed just by root
+            os.chmod(path, 0o600)
```

Comment:
Let's do the chmod everytime, that way if the file already exists but with relaxed permissions, we'll be sure to make it secure.
```suggestion
    # the file can be changed just by root
    os.chmod(path, 0o600)
```